hack/generate-examples

```diff
@@ -17,15 +17,15 @@ mkdir -p "$PATH_EXAMPLES"
 # render cloud-specific templates
 for cloud in aws azure gcp alicloud openstack local; do
   echo "* Rendering manifests for cloud '$cloud':"
-  for resource in 30-cloudprofile 40-secret-seed 50-seed 70-secret-cloudprovider 80-secretbinding-cloudprovider 90-shoot; do
+  for resource in 25-controllerinstallation 30-cloudprofile 40-secret-seed 50-seed 70-secret-cloudprovider 80-secretbinding-cloudprovider 90-shoot; do
     echo "  * Template '$resource' rendered."
     mako-render --var cloud="$cloud" "$PATH_TEMPLATES/$resource.yaml.tpl" > "$PATH_EXAMPLES/$resource-$cloud.yaml"
   done
   echo
 done
 
 # render cloud-independent templates
-for template in 05-project-dev 25-controllerregistration 25-controllerinstallation 60-quota 95-configmap-custom-audit-policy; do
+for template in 05-project-dev 25-controllerregistration 60-quota 95-configmap-custom-audit-policy; do
```

Comment:
Why have you done this change? The `ControllerInstallation` is cloud-independent. If you need it for the local setup please put the required `ControllerInstallation` resources into [this file](https://github.com/gardener/gardener/blob/master/hack/dev-setup-local) with

```bash
cat <<EOF | kubectl apply -f -
---
apiVersion: core.gardener.cloud/v1alpha1
...
EOF
```

Reply:
The `ControllerInstallation` contains a seed name which is right now `aws`, it refers to the seed created with `50-seed-aws.yaml`. Does this still qualify as "cloud-independent"? I could easily do what you suggest for the local setup, what about all other clouds?

Reply:
My opinion is that this https://github.com/gardener/gardener/blob/be4dc668089b0f84bece9f08f1a35f53f36c0b6a/example/25-controllerinstallation.yaml#L16 should not be `aws`, but `john-doe`, for example.

Reply:
Yes, agreed to @vpnachev. `aws` was just an example name, it can be anything. Maybe let's not use `john-doe` but `my-seed` or sth similar?

Reply:
OK, will do as suggested.